Tests/ReactiveSwiftTests/SignalSpec.swift

```diff
@@ -213,6 +214,97 @@ class SignalSpec: QuickSpec {
 			}
 		}
 
+		describe("interruption") {
+			it("should not send events after sending an interrupted event") {
+				let queue: DispatchQueue
+				let counter = Atomic<Int>(0)
+
+				if #available(macOS 10.10, *) {
+					queue = DispatchQueue.global(qos: .userInitiated)
+				} else {
+					queue = DispatchQueue.global(priority: .high)
+				}
+
+				let (signal, observer) = Signal<Int, NoError>.pipe()
+
+				var hasSlept = false
+				var events = [Event<Int, NoError>]()
```

Comment:
The format we normally use is separating type and value: `var events: [Event<Int, NoError>] = []`.